cvat-core/src/annotations-objects.ts

```diff
@@ -2187,6 +2187,11 @@ export class MaskShape extends Shape {
 
     constructor(data: SerializedShape, clientID: number, color: string, injection: AnnotationInjection) {
         super(data, clientID, color, injection);
+        const [left, top, right, bottom] = this.points.slice(-4);
+        const { width, height } = this.frameMeta[this.frame];
+        if (left > width || top > height || right > width || bottom > height) {
```

Comment:
left >= width? 

I suppose if we have image with width 1024px, the latest pixel has coordinate 1023 on X axis and left position 1024 is already out of the image

Reply:
The same for other coordinates

Reply:
applied